sherpa/sherpa.rc

```diff
@@ -1,10 +1,22 @@
 # SHERPA_VERSION 4.14.2
 [options]
-# Plotting packages available- pylab
-plot_pkg   : pylab dummy
+# Plotting backends (space separated list)
+# will be tried in order, the first one the imports is set as default
+# plotting backend. If no listed backend can be imported, Sherpa sets
+# a dummy backend as default.
+# The backend can be changed in a running sherpa session.
+# The following line would try all available backends:
```

Comment:
Does the following line 'try all backends' or does it find pylab and stop

Reply:
It stops at the first one that it find. That could be pylab (if installed) or not. Since dummy is always used as a fallback (even if not listed here) this line does not make much sense with just the pylab backend. However, we already have a PR that will add more backends, which will make this more useful.

So, I suggest to keep it in, because it's not wrong and will improve from "not wrong" to "actually useful" soon. 
I do push a change with a slight wording adjustment now.